release/air_tests/air_benchmarks/mlperf-train/resnet50_ray_air.py

```diff
@@ -481,15 +481,26 @@ def append_to_test_output_json(path, metrics):
             train_loop_config["data_loader"] = TF_DATA
         else:
             logger.info("Using Ray Datasets loader")
+
+            # Enable block splitting to support larger file sizes w/o OOM.
+            ctx = ray.data.context.DatasetContext.get_current()
+            ctx.block_splitting_enabled = True
+
             datasets["train"] = build_dataset(
                 args.data_root,
                 args.num_images_per_epoch,
                 args.num_images_per_input_file,
             )
+            # Set a lower batch size for images to prevent OOM.
+            batch_size = 32
```

Comment:
Can we specify this from the top-level script? IIRC we specify it as an arg already?

Reply:
Addressed offline - this batch size is unrelated to the one used by the trainer.